infra/cmake/packages/AbseilConfig.cmake

```diff
@@ -36,4 +36,9 @@ function(_Abseil_import)
   set(Abseil_FOUND TRUE PARENT_SCOPE)
 endfunction(_Abseil_import)
 
+set(CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS_DEBUG} -fPIC")
+set(CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} -fPIC")
+set(CMAKE_C_FLAGS_RELEASE   "${CMAKE_C_FLAGS_RELEASE} -fPIC")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fPIC")
+
```

Comment:
because of gpu_cl, Abseil lib are build as PIC not only for gpu_cl and other modules,
building pic. if gpu_cl requires as a PIC, hwo about building two kinds of libs on here (pic, shared) with different name and use it?

Reply:
oh... Thanks. I will check and modiy.